rules/go_rules.build_defs

```diff
@@ -598,14 +598,14 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
 
     # Now compile it in a separate step.
     cmd = [
-        'export GOPATH=$(find $TMP_DIR \( -name src -o -name pkg \) -exec dirname {} \; | sort | uniq | tr "\\n" ":") GO111MODULE="off"',
-        '$TOOLS_GO install -toolexec "$TOOLS_BUILDID" -gcflags "-trimpath $TMP_DIR" -asmflags "-trimpath $TMP_DIR" ' + ' '.join(all_installs or install),
+        'export GOPATH="$(find \"$TMP_DIR\" \( -name src -o -name pkg \) -exec dirname {} \; | sort | uniq | tr \'\\n\' \':\')" GO111MODULE="off"',
+        '"$TOOLS_GO" install -toolexec "\\\"$TOOLS_BUILDID\\\"" -gcflags "-trimpath \\\"$TMP_DIR\\\"" -asmflags "-trimpath \\\"$TMP_DIR\\\"" ' + ' '.join(all_installs or install),
```

Comment:
maybe use a raw string for this? think it'd be easier to read

Reply:
Yeah, I can see a few places I've missed as well, I'll commit as-is (and known working) and then go back for a tidy-up pass with all the other tools etc.